Client/Frontend/Browser/PreviewViewController.swift

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import Shared
+import Storage
+import WebKit
+
+class PreviewViewController: UIViewController {
+    
+    var openURLInNewTab: ((URL) -> Void)?
+    var copyURL: ((URL) -> Void)?
+    var shareURL: ((URL) -> Void)?
+    
+    private let tab: Tab
+    private(set) var url: URL
+    
+    override var previewActionItems: [UIPreviewActionItem] {
+        let openInNewTabAction = UIPreviewAction(title: Strings.OpenNewTabButtonTitle, style: .default) { previewAction, viewController in
```

Comment:
If `previewAction, viewController in` are not used you can replace it with `_, _ in`
Same goes for `copyAction` and `shareAction`